.eslintignore

```diff
@@ -1,5 +1,6 @@
 node_modules/*
-modules/*/js/*.js
+modules/*/js/*
+modules/*/js/*/*
```

Comment:
Can you add a comment explaining that eslint is being ignored on the compiled javascript because it's run on the uncompiled jsx directory? At least, I hope that's why this is here..

Reply:
done